composer.json

```diff
@@ -19,17 +19,18 @@
         }
     },
     "require": {
+        "php": "^7.1",
         "monolog/monolog": "^1.0.2",
         "zendframework/zend-modulemanager": "^2",
         "zendframework/zend-stdlib": "^2 || ^3",
         "zendframework/zend-servicemanager": "^2 || ^3"
     },
     "require-dev": {
         "container-interop/container-interop": "^1.0",
-        "jakub-onderka/php-parallel-lint": "^0.9",
-        "johnkary/phpunit-speedtrap": "^1.0",
-        "squizlabs/php_codesniffer": "^2.9",
-        "phpunit/phpunit": "^5|^4.8"
```

Comment:
What about phpunit v8?

Reply:
i believe phpunit 8 requires php72. i figured we can do that in a subsequent pull where you bump monolog. what do you think?

Reply:
It isn't problem. You can keep also 7.5 and prepare to smooth switch in future

Reply:
@snapshotpl here's the most recent php74 master build using latest deps:
https://travis-ci.org/enlitepro/enlite-monolog/jobs/632741435?utm_medium=notification&utm_source=github_status

php8 now throws several deprecation notices, which i see you've addressed in #57 

i was hoping you'd be interested in addressing those deprecations in a separate PR, so that we can clean up 2.x before we cut 3.0 : )